images/router/haproxy/conf/haproxy-config.template

```diff
@@ -155,8 +175,13 @@ backend be_edge_http_{{$cfgIdx}}
   mode http
   balance leastconn
   timeout check 5000ms
+  {{ if (eq $cfg.TLSTermination "") }}
+  cookie OPENSHIFT_{{$cfgIdx}}_SERVERID insert indirect nocache httponly
+  {{ else }}
+  cookie OPENSHIFT_EDGE_{{$cfgIdx}}_SERVERID insert indirect nocache httponly secure
+  {{ end }}
```

Comment:
Ultimately, no matter where we want to put a flag (on the router command or on a route by route basis) it is up to the underlying implementation to support sticky sessions which is under the administrator's control.  For the OOTB experience, it is always enabled.  We can decide to make that more flexible as we progress since it is additive.  If we make an api change to request sticky sessions we will, of course, be assuming that the underlying router implementation is honoring the request which we have no control over.